gems/smithy/lib/smithy/anvil/client/templates/specs/endpoint_provider_spec.erb

```diff
@@ -6,6 +6,19 @@ require_relative '../spec_helper'
 
 module <%= namespace %>
   describe EndpointProvider do
+    # TODO: Can be replaced by stub_responses config once implemented
+    let(:stub_send) do
+      Class.new(Smithy::Client::Plugin) do
+        def add_handlers(handlers, _config)
+          handlers.add(Class.new(Smithy::Client::Handler) do
```

Comment:
HandlerBuilder has a handle method that accepts a block already, so you can skip the add_handlers call.

Reply:
Updated.